esf_main.tex

```diff
@@ -207,33 +207,74 @@ \subsubsection{Position in car}
-TODO
+The IMD used will be a Bender A-ISOMETER IR155-3204. The output is normally high and only low if it does not detect a ground fault. The output is then sent to the IMD latch board, where it is powers a SPST-NO relay which closes a switch in the shutdown circuit. The output of the IMD is also monitored directly by an Atmega on the AIR control board. The status of the shutdown circuit is monitored by the MSP board. If the MSP board detects the relay is  open it activates the IMD light. The IMD has a start up time of $\leq$ 2s.
+%Describe the IMD used and use a table for the common operation parameters, like supply voltage, set point, etc. Also describe how the IMD indicator light is wired, etc.
+
 
+\begin{center}
 	\begin{table}[H]
-	    \centering
-	    \begin{tabular}{|l|l|}
-	    \hline
-	        Supply voltage range: & 10...36VDC \\ \hline
-	        Supply voltage & 12VDC \\ \hline
-	        Environmental temperature range: & Unknown \\ \hline
-	        Selftest interval: & Every 5 minutes \\ \hline
-	        High voltage range: & 0-1000 VDC \\ \hline
-	        Set response value: & 100 k\ohm TODO:double check this\\ \hline
-	        Max. operation current: & 150 mA \\ \hline
-	        Approximate time to shut down at 50$\%$ of the response value: & $\leq$ 40 sec \\ \hline
-	    \end{tabular}
-	    \caption{Parameters of the IMD}
-	    \label{IMDparameters}
+		\begin{tabular}{|l|l|}
+			\hline
+			Supply voltage range: &  10..36VDC \\
+			\hline
+			Supply voltage: &  12VDC\\
+			\hline
+			Environmental temperature range: &  -40..105$^{\circ}$C \\
+			\hline
+			Selftest interval: &  Always at startup, then every 5 minutes \\
+			\hline
+			High voltage range: &  DC 0..1000V \\
+			\hline
+			Set response value: &  100k$\Omega$ \\
+			\hline
+			Max. operation current: &  150mA \\
+			\hline
+			Approximate time to shut down at 50\% of the response value:&  $\leq$ 40s \\
+			\hline
+		\end{tabular}
+		\caption{Parameters of the IMD}
+		\label{IMDParameters}
 	\end{table}
+\end{center}
 
-\subsubsection{Wiring/cables/connectors/}
+
+
+
+
+\subsubsection*{2.2.2 Wiring/cables/connectors/}
+The  IMD uses the TYCO-MICRO MATE-N-LOK 1 x 2-1445088-8 connector and it's mate. It connects to the accumulator interface board, which sends the signal to a through a 28 AWG ribbon cable to the IMD latch using a 6-pin TE Connectivity MICRO MATCH 338068-6 connector.The 28 AWG wire is used because the current draw of the IMD is 150mA, which allows high gauge wire to be used. There is a 1 amp fuse protecting the high voltage wiring of the IMD. % I don't know the details of the fuse, but it is mentioned in both last year's ESF and in the ESF writing guide. It is visible in some of the plans for the electrical system. I need to know where I can find the details. Last year it was a 1 amp fuse, but it might be different this year. I am writing 1 amp for now, but it needs to be confirmed.
```

Comment:
As per last year's ESF, please provide a \ref to a section you create in the appendix for the IMD datasheet with the important parameters (and a hyperlink to the full datasheet), as well as a \ref to a section in the appendix where the fuse information will go. I will fill in the fuse information later today.